packages/design-system/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -24,6 +26,45 @@ export class HelpDrawer extends React.PureComponent {
 
   componentDidMount() {
     if (this.headingRef) this.headingRef.focus();
+
+    const eventAction = 'onComponentDidMount';
+
+    /* Send analytics event for helpdrawer open */
+    this.sendAnalyticsEvent(eventAction);
+  }
+
+  componentWillUnmount() {
+    const eventAction = 'onComponentWillUnmount';
+
+    /* Send analytics event for helpdrawer close */
+    this.sendAnalyticsEvent(eventAction);
+  }
+
+  sendAnalyticsEvent(eventAction) {
+    const analyticsOverrides = get(this.props.analytics, eventAction);
+    const analyticsDisabled = analyticsOverrides === false;
+
+    if (window.utag && !analyticsDisabled) {
+      const payload = merge(this.getDefaultAnalytics(eventAction), analyticsOverrides);
+      sendTealiumEvent(payload);
+    }
+  }
+
+  getDefaultAnalytics(eventAction) {
```

Comment:
This doesnt have to be a component method, it can also just be an object. I recommend defining this at the top of the file like so:
```
const defaultAnalytics = {
   ....
}
```

This way usage can just be `get(defaultAnalytics, eventAction)`